### ISSUE TITLE
using hmac-sha256

### ISSUE THREAD

**Issue by ShiraRotholz** [bettiolo/oauth-signature-js#45]
Hi,
I am trying to use oauth_signature_method HMAC-SHA256, but it seems like its not supported (Auth fails).
Can you please explain how to use this?

Thanks,

**Comment by FranklynSeabraOSF** [comment]
I don't think this lib supports SHA256, I also need it because NetSuite does not support SHA1 anymore. But I'll have to implement as it seems. @ShiraBoa
